app/src/main/java/com/example/dineDelight/pages/UserReservationsScreen.kt

```diff
@@ -0,0 +1,65 @@
+package com.example.dineDelight.pages
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.automirrored.filled.ArrowBack
+import androidx.compose.material3.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.navigation.NavController
+import com.example.dineDelight.models.Reservation
+import com.example.dineDelight.repositories.ReservationRepository
+import com.google.firebase.auth.FirebaseAuth
+
+@Composable
+fun UserReservationsScreen(navController: NavController) {
+    val userId = FirebaseAuth.getInstance().currentUser?.uid.orEmpty()
+    val reservations by ReservationRepository.reservations.collectAsState()
+
+    LaunchedEffect(userId) {
+        ReservationRepository.getUserReservations(userId)
+    }
+
+    Column(
+        modifier = Modifier.fillMaxSize().padding(16.dp),
+        verticalArrangement = Arrangement.Top
+    ) {
+        // Back button
+        IconButton(onClick = { navController.popBackStack() }) {
+            Icon(imageVector = Icons.AutoMirrored.Filled.ArrowBack, contentDescription = "Back")
+        }
+
+        Text(text = "Your Reservations", style = MaterialTheme.typography.titleLarge)
```

Comment:
I think we can keep this title as "My Reservations".

```suggestion
        Text(text = "My Reservations", style = MaterialTheme.typography.titleLarge)
```